e2e/Tests/prerequisites/e2eTestsSetup.ps1

```diff
@@ -773,7 +773,7 @@ catch
 CleanUp-Certs
 
 # Creating a file to run to load environment variables
-$loadScriptDir = Join-Path $PSScriptRoot "..\..\..\..\.." -Resolve
```

Comment:
The file path was refactored previously, so correcting this

Reply:
It needs to go up to the directory above the repo root.

So if this file is in e2e/Tests/prerequisites, this should go up 3 dirs, so "..\..\..".